concepts/nil/about.md

```diff
@@ -1,30 +1,67 @@
-# Zero Values
+# Introduction
 
-Unlike some other languages, Go has no empty/null/undefined state for uninitialized variables. Every type has a [zero value][zero_values] that uninitialized variables will hold upon declaration.
+Go does not have a concept of undefined for variable values. Variables declared without an explicit initial value default to the [zero value][zero_values] for their respective type.
 
-These default values are called the zero values for their respective types:
+The zero value for primative types such as booleans, numeric types, and strings are `false`, `0`, and `""`, respectively.
+
+The identifier `nil`, meaning zero, is the zero value for more complex types such as pointers, interfaces, channels, maps, slices, and functions.
+
+The following table details the zero value for Go's types.
 
 | Type      | Zero Value |
 | --------- | ---------- |
 | bool      | false      |
 | numeric   | 0          |
 | string    | ""         |
 | pointer   | nil        |
-| func      | nil        |
 | interface | nil        |
-| slice     | nil        |
 | channel   | nil        |
 | map       | nil        |
+| slice     | nil        |
+| func      | nil        |
 
-The predeclared identifier `nil`, meaning zero, is the zero value for the more complex types in Go. Comparing a type whose zero value is not `nil` to `nil`, like string, is an error:
+## Zero Value Construction
+
+The `var` keyword can be used to construct any type to its zero value:
 
 ```go
-    func main() {
-        myString := "exercism"
-        if myString != nil { // invalid operation: myString != nil (mismatched types string and nil)
-            fmt.Println(myString)
-        }
-    }
+func main() {
+  var myBool bool
+  fmt.Println(myBool)
+}
+```
+
+```go
+func main() {
+  var myMap map[int]int
+  fmt.Println(myMap)
+}
+```
```

Comment:
I would probably leaving this out, as this document is about `nil` and not about zero values. If you'd like to link between concepts, I'd suggest using the Concept Widget as described in [this document](https://exercism.org/docs/building/markdown/widgets).